### DISCUSSION TITLE
Feature request: Custom browser parameters

### DISCUSSION THREAD

**Discussion by shoguevara** [will-stone/browserosaurus#437] [Ideas]
Hi! Thanks for the app! It's a great productivity tool I now can't imagine my life without!

But could you please let configuring custom browser launch command? I'll explain where it is coming from: I'm using different profiles for my Chrome Browser - for work and a personal one. АТМ when I'm choosing to open the link with Chrome it gets sent to whatever browser profile is active in foreground. It annoys a bit, especially when I have to copy-paste URL from the wrong profile top the correct one

**Comment by will-stone** [comment]
Hi @shoguevara, it's certainly possible but not something I need right now and would be a lot of work and therefore maintenance. Off the top of my head, it would require:

- Input row(s) added to the Tiles section of Preferences.
- Logo selector (selection of in-built and/or own custom _uploader_).
- Storage of configuration.
- Error handling: what happens if you set a bad command?
- Command parsing: what happens if someone using your computer decides to set the command as `rm -rf /`?
- Testing.

If you reckon that all sounds doable, please fork the project and I'd be happy to see a PR, but please bear in mind that as I'll be the one left ~holding the baby~ maintaining the project, it'll only be included in Browserosaurus if it looks manageable.

Alternatively, I highly recommend [Finicky](https://github.com/johnste/finicky) which allows you to set commands and rules. You can even have B pass URLs to F, and vice versa.

**Reply by shoguevara** [reply]
Thanks a lot for your answer!
I get it - it's OK, it doesn't cancel my gratitude in any way. If I had time/expertise in the filed, this request would've ben as a merge request)
And separate thanks for the suggestion!)

**Reply by rbreaves** [reply]
Love this app as well, but like shoguevara being able to use profiles would be amazing. I did not know about Finicky but I may use that. It is no longer maintained though. I may also look at possibly creating a PR for this one if I do not use Finicky going forward or trying to maintain it.

**Reply by alansuspect** [reply]
I was just about to create a new feature request with this option then I found this thread. My preference would be for Firefox so I don't know if it would involve even more work for diferent browsers?

**Reply by neo2049** [reply]
This would be amazing if this could happen.
